eks/metrics-server/metrics-server.go

```diff
@@ -3,455 +3,56 @@
 package metricsserver
 
 import (
-	"context"
-	"errors"
 	"fmt"
-	"io"
-	"reflect"
-	"strings"
-	"time"
 
-	eks_tester "github.com/aws/aws-k8s-tester/eks/tester"
 	"github.com/aws/aws-k8s-tester/eksconfig"
-	"github.com/aws/aws-k8s-tester/pkg/fileutil"
-	k8s_client "github.com/aws/aws-k8s-tester/pkg/k8s-client"
-	"github.com/aws/aws-k8s-tester/pkg/timeutil"
-	"github.com/aws/aws-sdk-go/aws"
-	"go.uber.org/zap"
-	apierrs "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/utils/exec"
+	k8sclient "github.com/aws/aws-k8s-tester/pkg/k8s-client"
+	gotemplate "github.com/aws/aws-k8s-tester/pkg/util"
 )
 
-// Config defines Dashboard configuration.
-type Config struct {
-	Logger    *zap.Logger
-	LogWriter io.Writer
-	Stopc     chan struct{}
-	EKSConfig *eksconfig.Config
-	K8SClient k8s_client.EKS
+// MetricsServer is an addon that installs Metrics Server
+type MetricsServer struct {
```

Comment:
Easiest way to maintain backwards compat is by putting this new logic into metrics-server/addon.go and reverting this metrics-server.go file to what it was before.